templates/terraform/examples/vpn_tunnel_beta.tf.erb

```diff
@@ -0,0 +1,61 @@
+resource "google_compute_vpn_tunnel" "tunnel1" {
```

Comment:
```suggestion
resource "google_compute_vpn_tunnel" "tunnel1" {
  provider = "google-beta"
  
```

You'll need to set the provider on every resource + add a google-beta block like https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/examples/managed_ssl_certificate_basic.tf.erb#L88 or the test should fail (and if it doesn't, that's a bug)

Reply:
I'm not sure that's true, since they aren't in the current generated test file: https://github.com/terraform-providers/terraform-provider-google-beta/blob/master/google-beta/resource_compute_vpn_tunnel_generated_test.go

Reply:
It's required for beta-only resources, or for `min_version` `beta`

Reply:
is that just to enforce its provider is beta? Why is it enforced for versioned tests but not added to tpg-beta tests when min_version isn't set? I did make the change, am just curious.

Reply:
> is that just to enforce its provider is beta

Correct! The tl;dr is that we need example tests with beta features to use the `google-beta` provider alias because the same config generates our OiCS examples, and they need to use `google-beta`. In the beta provider tests normally, `google` is `google-beta` in disguise, but there's no way [that I know of] to perform the same trick in our OiCS samples.

See this thread for some context: https://github.com/GoogleCloudPlatform/magic-modules/pull/1464#discussion_r262197684